oktapam/resource_sudo_commands_bundle_test.go

```diff
@@ -0,0 +1,57 @@
+package oktapam
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/okta/terraform-provider-oktapam/oktapam/client"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+	"github.com/okta/terraform-provider-oktapam/oktapam/logging"
+	"github.com/okta/terraform-provider-oktapam/oktapam/utils"
+)
+
+func TestAccResourceSudoCommandsBundle(t *testing.T) {
+	checkTeamApplicable(t, true)
+	resourceName := "oktapam_sudo_commands_bundle.test_acc_sudo_commands_bundle"
+	sudoCommandsBundleName := fmt.Sprintf("test-sudo-commands-bundle-%s", randSeq())
+
+	resource.Test(t, resource.TestCase{
```

Comment:
we generally try and run both a create and update for resources when doing a test.